services/gateway-api/src/circuit_breaker/health_monitor.py

```
@@ -0,0 +1,450 @@
+    async def check_service_health(self, service_name: str) -> HealthCheckResult:
+        """
+        Perform a single health check for a service
+
+        Args:
+            service_name: Name of the service to check
+
+        Returns:
+            HealthCheckResult with status and metrics
+        """
+        if service_name not in self._health_checks:
+            raise ValueError(f"Service '{service_name}' not registered")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)